### ISSUE TITLE
Chocolatey Only Installs V1.17 and with Traffmonetizer

### ISSUE THREAD

**Issue by MrIncUK** [Genymobile/scrcpy#3504]
- [ ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ ] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: [e.g. Debian, Windows, macOS...]  WINDOWS 10
 - scrcpy version: [e.g. 1.12.1]   1.17
 - installation method: [e.g. manual build, apt, snap, brew, Windows release...]  Installed via Chocolatey in Powershell.  
 - device model:  Nvidia TV Shield PRO
 - Android version: [e.g. 10]  Android TV Latest

**Describe the bug**
It looks like SCRCPY installs a Monetizer called Traffmonetizer and interferes with the operation of the TV Shield.

Whilst watching Netflix of Amazon Prime or running Emby, the screen will flash and go white and the main home screen appears, flash white again a couple of times again and display the Traffmonetizer logo. I was able to to take a short video of this and then grab a capture of the logo which allowed me to trace the problem.  Sometimes the screen will disappear and I am returned to what I am watching, other times the app will abort and I remain on the home screen.
This is an example from using the Youtube app.
https://youtu.be/SwitH82Qsjw

```
Please paste terminal output in a code block.
This is the output from 'installing' via Chocolatey:
`
PS C:\WINDOWS\system32> choco install scrcpy
Chocolatey v0.10.15
Installing the following packages:
scrcpy
By installing you accept licenses for the packages.
scrcpy v1.17 already installed.
 Use --force to reinstall, specify a version to install, or try upgrade.

Chocolatey installed 0/1 packages.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Warnings:
 - scrcpy - scrcpy v1.17 already installed.
 Use --force to reinstall, specify a version to install, or try upgrade.
PS C:\WINDOWS\system32>
[chocolatey.log](https://github.com/Genymobile/scrcpy/files/9595525/chocolatey.log)
`

```

Please do not post screenshots of your terminal, just post the content as text instead.
See youtube video of the problem with the Traffmonetizer that got pushed onto the Shield with Scrcpy.  I know this is the culprit because I did a factory reset on the Shield to remove the problem assuming it was in a store app, but after the reset I did not instrall any apps off the store and only re-implemented the Developer mode, Networking debugging and installed Scrcpy via ADB.

**Comment by rom1v** [comment]
Hi,

> It looks like SCRCPY installs a Monetizer called Traffmonetizer

No, it does not. At least, the official release does not do that.

The latest scrcpy version in chocolatey is currently [1.24](https://community.chocolatey.org/packages/scrcpy). It's unexpected that you don't get the latest version (try to uninstall/reinstall the package to get the latest version, or ask for help in their forums, I don't really know chocolatey).
This is an unofficial package, but looking at the [script](https://github.com/n3rd4i/scrcpy/blob/master/tools/chocolateyinstall.ps1) it downloads the official release from GitHub.

> See youtube video of the problem with the Traffmonetizer that got pushed onto the Shield with Scrcpy.

This is not pushed by scrcpy.

> I know this is the culprit because I did a factory reset on the Shield to remove the problem assuming it was in a store app, but after the reset I did not instrall any apps off the store and only re-implemented the Developer mode, Networking debugging and installed Scrcpy via ADB.

You might (for example) have a malware on your desktop that installs this malware app as soon as you get an adb connection. Try from another (trusted) computer.

**Comment by MrIncUK** [comment]
Thanks for that suggestion.  Looks like a problem on my PC.  I factory reset the Shield and I installed Choco and Scrcpy on a clean VM of Windows 10.  It installed 1.24 with no issues.  I got it to work with Shield fine and I watched an hour's program on Prime Video without any issues surfacing,  I will uninstall on my PC and do another system scan as I have subscription to Spybot S&D.

So problem resolved I think!
